### PR TITLE
Doc/token integration

### PR DESCRIPTION
**Pull request by arnigudj**
documentation improvements. 
No changes to contracts or any functionality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,19 +4,133 @@
...
+| QR code  | ![ISK](docs/0xc642549743a93674cf38d6431f75d6443f88e3e2.png)  |
+| Logo | ![ISK](assets/tokens/isk/logo.png)  |
+
+
+## Test tokens
+
+For the innovators, product builders, and other curious minds we've opened up a sandboxed version of our system that is connected to the Rinkeby test network. 
+Play around with our money in a safe environment to understand how this fits into your platform. [Go to sandbox](https://sandbox.monerium.dev/).
+
+Please fill out the "Partner with Monerium" form on [monerium.com](https://monerium.com) to get test money for the Ropsten or Kovan test networks.
```
Comment (gislik) [inline README.md:74]: We should maybe link directly to https://monerium.com/#partners

File: README.md [hunk 2]
```diff
@@ -4,19 +4,133 @@
...
+| GBP      | 0x01df10e345d0364d3a5b8422a66af6305803bd1e                                          | 0xd9979346224e7b147caddf399b56357e20d3e67c                                          | 0xe28884ed5bd43e3f9f1dd733d254c9f5c6f983d2                                          |
+| ISK      | 0x0c9d7a0d8bf4bc9d15f577bbf650ebc8044a71db                                          | 0x80b02ef56cbbc542f0ce89ad1d2a680244da9a63                                          | 0x39ad1ad871787ba4b3df5b8ac3d81b2c9b7c6290                                          |
+| USD      | 0x09c0a236e1227500f495cb0731c4af69b49639a5                                          | 0x3781dcdd60e006e33b664dce0d6be934f0a139c8                                          | 0x57724f65b3f914de7820c6f76b2099fa3a90f509                                          |
+
+## Using Monerium money / Token integration
+
+We at [Monerium](https://monerium.com) er really exited to talk to all developers and builders that want to use money in their applications.
```
Comment (gislik) [inline README.md:86]: Spelling error... We at Monerium are ...

File: README.md [hunk 3]
```diff
@@ -4,19 +4,133 @@
...
+Go to [monerium.com](https://monerium.com) and fill out the "Partner with Monerium" form and our expert staff
```
Comment (gislik) [inline README.md:87]: https://monerium.com/#partners

File: README.md [hunk 4]
```diff
@@ -4,19 +4,133 @@
 ![GitHub release](https://img.shields.io/github/release/monerium/smart-contracts.svg)
 [![Build Status](https://travis-ci.com/monerium/smart-contracts.svg)](https://travis-ci.com/monerium/smart-contracts)
 
-The [Monerium](https://monerium.com) e-money platform offers programmable fiat money on blockchains, an indispensable building block for the nascent blockchain economy.
+The [Monerium](https://monerium.com) e-money offers programmable fiat money on blockchains, an indispensable building block for the nascent blockchain economy.
+
+Monerium EMI is an Electronic Money Institution, currently the only financial entity licenced to issue e-money on blockchains. 
+Electronic money (e-money) has been recognized in the European Economic Area (EEA) as a digital alternative to cash since 2000 when the first e-money Directive was introduced. Monerium e-money is 1:1 backed in fully segregated, unencumbered, high-quality liquid assets and unconditionally redeemable on demand. [Read more about e-money here](https://monerium.com/monerium/2019/06/28/e-money-the-digital-alternative-to-cash.html).
```
Comment (gislik) [inline README.md:10]: Purely cosmetic but I'd prefer:

Read more about e-money [here](https://monerium.com/monerium/2019/06/28/e-money-the-digital-alternative-to-cash.html)

File: README.md [hunk 5]
```diff
@@ -4,19 +4,133 @@
...
+Monerium e-money tokens [v1.0.3](https://github.com/monerium/smart-contracts/releases/tag/v1.0.3) are live on Ethereum mainnet and are [ERC20](https://eips.ethereum.org/EIPS/eip-20) and [ERC677](https://github.com/ethereum/EIPs/issues/677) compliant.
+
+### Euro / EUR
+
```
Comment (gislik) [inline README.md:17]: Cosmetics again:

Logos and QR codes should be 1/2 of this size and aligned center in the cell.

File: README.md (hunk 2 above)
Comment (gislik) [reply README.md:86]: Spelling error... really excited ...

File: README.md [hunk 6]
```diff
@@ -4,19 +4,133 @@
...
+  // on the recipient [ERC677].
+  transferAndCall(to, amount, data)
+
+  // Returns the total supply.
+  totalSupply()
+
+  // Returns the number tokens associated with an address.
+  balanceOf(who)
+
+  // Returns the allowance for a spender
```
Comment (gislik) [inline README.md:114]: Missing trailing dot for consistency

File: assets/tokens/isk/info.json [hunk 7]
```diff
@@ -0,0 +1,56 @@
+{
+    "name": "Monerium EUR emoney",
```
Comment (gislik) [inline assets/tokens/isk/info.json:2]: Name does not match directory name

File: assets/tokens/usd/info.json [hunk 8]
```diff
@@ -0,0 +1,56 @@
+{
+    "name": "Monerium EUR emoney",
```
Comment (gislik) [inline assets/tokens/usd/info.json:2]: Name does not match filename

File: assets/tokens/isk/info.json [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+    "symbol": "EURe",
```
Comment (gislik) [inline assets/tokens/isk/info.json:3]: Symbol does not match directory name

File: assets/tokens/isk/info.json [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+    "ticker": "EUR",
```
Comment (gislik) [inline assets/tokens/isk/info.json:4]: Ticker does not match directory name

File: assets/tokens/isk/info.json [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+    "networks": {
+        "1" : {
+            "name": "Ethereum Mainnet",
+            "short_name": "eth",
+            "chain": "ETH",
+            "network": "mainnet",
+            "chain_id": 1,
+            "network_id": 1,
+            "contract_address": "0xc642549743a93674cf38d6431f75d6443f88e3e2",
+            "ens": "eur.monerium.com",
```
Comment (gislik) [inline assets/tokens/isk/info.json:18]: ens does not match directory name

File: assets/tokens/usd/info.json [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+    "symbol": "EURe",
```
Comment (gislik) [inline assets/tokens/usd/info.json:3]: Symbol does not match directory name

File: assets/tokens/usd/info.json [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+    "ticker": "EUR",
```
Comment (gislik) [inline assets/tokens/usd/info.json:4]: Ticker does not match directory name

File: assets/tokens/usd/info.json [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+    "networks": {
+        "1" : {
+            "name": "Ethereum Mainnet",
+            "short_name": "eth",
+            "chain": "ETH",
+            "network": "mainnet",
+            "chain_id": 1,
+            "network_id": 1,
+            "contract_address": "0x3231cb76718cdef2155fc47b5286d82e6eda273f",
+            "ens": "eur.monerium.com",
```
Comment (gislik) [inline assets/tokens/usd/info.json:18]: ens does not match directory name

**Review by gislik — CHANGES_REQUESTED** [review]
There are a few mistakes that need to be fixed before we release.

File: README.md (hunk 5 above)
Comment (arnigudj) [reply README.md:17]: I'll make it smaller, but I disagree with center alignment.

File: assets/tokens/isk/info.json (hunk 7 above)
Comment (arnigudj) [reply assets/tokens/isk/info.json:2]: Good catch

**Review by gislik — APPROVED** [review]

**Pull request merged**